bots/tests-scan

```diff
@@ -288,8 +329,22 @@ def cockpit_tasks(api, update, contexts, repo):
                 branch_contexts[branch] = [ ]
             branch_contexts[branch].append(context)
 
+    pulls = []
+    if pull_number:
+        # Avoid processing pull requests with the same number on external
+        # repositories: if repo is None, the api is querying the repo the pull
+        # request belongs to
+        if repo is None:
+            pull = api.get("pulls/{0}".format(pull_number))
+            if pull:
+                pulls.append(pull)
+            else:
+                sys.stderr.write("Can't find pull request {0}\n".format(pull_number))
```

Comment:
This needs to exit here, otherwise it's going to behave strangely.

Reply:
well technically it won't have any pulls to scan, so it should do nothing; but i'll explicitely exit there.